pkg/apis/core/types_shoot.go

```diff
@@ -1205,6 +1207,18 @@ type MaintenanceAutoUpdate struct {
 	MachineImageVersion *bool
 }
 
+// MaintenanceAutoRotation contains information about which credentials should be automatically rotated.
+type MaintenanceAutoRotation struct {
+	// ETCDEncryptionKey indicates whether the etcd encryption key may be automatically rotated (default: false).
```

Comment:
Same for the others.

```suggestion
	// ETCDEncryptionKey indicates whether the etcd encryption key should be automatically rotated (default: false).
```